cdntaxreceipts.functions.inc

```diff
@@ -738,7 +738,7 @@ function cdntaxreceipts_issueTaxReceipt($contribution, &$collectedPdf = NULL, $m
   if ( $contributiontype->name == 'In-kind' ) {
```

Comment:
Wouldn't this now read  if ( $contributiontype->name == 'In-kind'  ||  $contributiontype->name == 'In Kind')